doc/tutorials/image/image.txt

```diff
@@ -0,0 +1,210 @@
+/*!
+@file image.txt
+@author Mehul Kumar Nirala
+@brief Tutorial for how to load and save images in mlpack.
+
+@page imagetutorial Image Utilities tutorial
+
+@section intro_imagetut Introduction
+
+Image dataset are becoming increasingly popular in deep learning.
+
+mlpack currently implements saving and loading of image. It uses [stb/](https://github.com/nothings/stb) to achieve this.

```

Comment:
Perhaps we should alter the wording here to something like: "mlpack's image saving/loading functionality is based on [stb/](https://github.com/nothings/stb)." Let me know what you think.